package.json

```diff
@@ -62,6 +63,14 @@
       "env": {
         "AMO_CDN": "https://addons-dev-cdn.allizom.org",
         "PROXY_API_HOST": "https://addons-dev.allizom.org",
+        "FXA_CONFIG": "default",
+        "CSP": false,
+        "NODE_APP_INSTANCE": "amo"
+      }
+    },
+    "amo:ui-tests": {
+      "command": "better-npm-run start-dev-proxy",
+      "env": {
         "FXA_CONFIG": "local",
```

Comment:
Can you remove this entry? and also `CSP` below. It is better to use the config file.

Reply:
You can add `NODE_CONFIG_ENV: ui-tests` though.

Reply:
What? Then why having `FAX_CONFIG` set here?